locales/templates/app.pot

```diff
@@ -495,6 +495,16 @@ msgid ""
 "wants to access your location."
 msgstr ""
 
+#. A checkbox option in the dialog which ask users to give geolocation
+#. permission to the site.         Users' decision will be remembered and
+#. future display of the dialog will be skipped if this         option is
+#. checked. These memorized decisions will be wiped out if the App is
+#. restarted.         %1$s is the App name (Firefox Rocket).
+#, c-format
+msgctxt "geolocation_dialog_message_cache_it"
+msgid "Don't ask again for this site until %1$s restarts"
```

Comment:
Proper quotes are smart quotes for strings: ’

Reply:
I think we already have this one in `my_shot_on_boarding_category_message`:https://pontoon.mozilla.org/id/firefox-rocket/app.po/?search=my_shot_&string=189499 seems fine on Pontoon?

Reply:
Ouch. That means it fell through the cracks. We should fix that, if possible (can be in a later export, we just need to remember about it)

Reply:
I can do it since we're going to fix it anyway :)

Reply:
Given it's not a blocking change... :P

Reply:
I'll fix both of them now.